lib/liquid/context.rb

```diff
@@ -27,7 +27,7 @@ def initialize(environments = {}, outer_scope = {}, registers = {}, rethrow_erro
       @this_stack_used = false
 
       if rethrow_errors
-        self.exception_handler = ->(e) { raise }
+        self.exception_handler = ->(e) { raise e }
```

Comment:
Hm I actually think it would be more intuitive to interrupt the filter chain. Why do you think it's better not to?